src/core/Canvas.js

```diff
@@ -14,6 +14,10 @@ export class Canvas extends BaseCanvas {
         this.createUiGroup();
     }
 
+    getCurrentScale () {
+        return +this._viewport.getCTM().a.toFixed(5);
```

Comment:
I think the function should return the number and not a string, leaving the transformation to the caller.

Reply:
It is returning a number. I've found this hack on stackoverflow an never saw it before.

Reply:
Oh you are right, I missed the `+` :wink: 
I still don't really get why this is needed?

Reply:
> I still don't really get why this is needed?

What do you mean? The number type cast? Thats to remove the trailing 0 at the end. Because with toFixed you get a String with n decimal places. The cast is mainly for the comparison in the Tests and it can check conditions like this: `canvas.getCurrentScale() === 0.5`

Reply:
> The cast is mainly for the comparison in the Tests and it can check conditions like this: `canvas.getCurrentScale() === 0.5`

Yes that's what i meant, thanks. IMO a function like this should return the raw number. If the caller (the tests in this case) need it a certain way, it should be their responsibility to transform it ...

But I'm fine with leaving it like this for now.

Reply:
`getCTM().a` was a product some weird JavaScript floating numbers.... This can only lead to error. I'll take the 5 decimal places, so the caller know its 5 and not something like this: 0.000000000000001

Reply:
Yes I understand the intention. I'm not sure about the conclusion that casting floats to strings and returning them to floats will lead to less errors than just leaving them as is but whatever.